apps/backend/src/events/dto/create-event.dto.ts

```diff
@@ -1,5 +1,12 @@
 import { OmitType } from '@nestjs/swagger';
+import { IsArray, IsOptional, ValidateNested } from 'class-validator';
+import { CreateDrinkActionWithoutEventIdDto } from 'src/drink-action/dto/create-drink-action-without-eventid.dto';
 
 import { Event } from '../entities/event.entity';
 
-export class CreateEventDto extends OmitType(Event, ['id', 'createdAt', 'ownerId']) {}
+export class CreateEventDto extends OmitType(Event, ['id', 'createdAt', 'ownerId']) {
+  @IsArray()
+  @ValidateNested({ each: true })
+  @IsOptional()
+  drinkActions?: CreateDrinkActionWithoutEventIdDto[];
+}
```

Comment:
I was wrong, the `@Type(() => CreateDrinkActionWithoutEventIdDto)` is required, class validator doesn't recognise this field otherwise